### ISSUE TITLE
An error occurred during the training

### ISSUE THREAD

**Issue by royal-dargon** [yongliang-wu/NumPro#9]
Is there a problem if the following phenomena occur during training.
25.35s/it]Error at loading 142473: list index out of range
Error at loading 201544: list index out of range
Error at loading 162858: list index out of range
Error at loading 200791: list index out of range
Error at loading 106706: list index out of range
{'loss': 343.127, 'grad_norm': 442.24884033203125, 'learning_rate': 3.367003367003367e-06, 'epoch': 0.01}                                                                                                                                                                                               
  0%|▍                                                                                                                                                                                                                                                             | 10/5928 [04:17<41:57:07, 25.52s/it]Error at loading 236319: list index out of range
Error at loading 247525: list index out of range
Error at loading 107189: list index out of range
Error at loading 169501: list index out of range
Error at loading 152490: list index out of range

**Comment by yongliang-wu** [comment]
This might be expected, as some video data could be unavailable. To handle this, I implemented a sample resampling strategy in `longva/train/train.py`, lines 1024–1026:

```python
except Exception as e:
    print(f"Error at loading {i}: {e}")
    return random.choice(self)
```

Was your training able to proceed normally after this point?
